03-task-manager/starter/db/connect.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose')
+
+
+
+
+const connectDB = (url) => {
+    return mongoose.connect(url)
```

Comment:
Nitpick:
I believe you may see some warnings in your terminal about certain mongoose features being disabled or deprecated. You can add this mongoose options object to fix those warnings:
```js
return mongoose.connect(url, {
  useNewUrlParser: true,
  useCreateIndex: true,
  useFindAndModify: false,
  useUnifiedTopology: true,
});
```

Reply:
I have the latest version so I didn't need to do that.